torch_frame/data/multi_embedding_tensor.py

```diff
@@ -196,3 +206,32 @@ def cat(
             # which is inconsistent with when dim=0
             offset = torch.LongTensor(offset_list)
             return MultiEmbeddingTensor(num_rows, num_cols, values, offset)
+
+    def _normalize_index(
+        self,
+        index: Union[int, Tensor],
+        dim: int,
+        is_slice_end: bool = False,
```

Comment:
We already have `_normalize_index` in `MultiNestedTensor`. https://github.com/pyg-team/pytorch-frame/blob/master/torch_frame/data/multi_nested_tensor.py#L373-L409

What's the difference?

Reply:
It's exactly the same. I'm moving it to the parent class

Reply:
Can we instead move `_normalize` to `_MultiTensor`, so that it can be used for both classes?

Reply:
Super, thanks.

Reply:
Done! PTAL :)